source/voice_control/s3_box_voice_assistant.markdown

```diff
@@ -206,3 +207,16 @@ Before you can use this device with Home Assistant, you need to install a bit of
      - Disable **LCD Backlight**.
 
       ![Toggle to enable/disable wake word](/images/assist/s3-box-disable-screen.png)
+
+## Updating the software on the S3-BOX
+
+To update the software on your S3-BOX, follow the steps below that reflect your setup.
+
+- **Option 1**: You have Home Assistant 2024.7 or later, and have not manually altered your ESPHome configuration for the S3-BOX:
+  - Once an update is available, you will receive an update notification, just like any other update.
+  - To install the precompiled new firmware directly on your box, make sure the S3-BOX is connected to your network, and under **ESP32 S3 BOX...Firmware**, select **Install**.
+- **Option 2**: You have Home Assistant 2024.6 or older, and have not manually altered your ESPHome configuration for the S3-BOX:
+  - Follow steps 1-3 of the procedure on [installing the software onto the S3-BOX](#installing-the-software-onto-the-esp32-s3-box).
+    - This installs the latest, precompiled firmware for your S3-BOX.
+- **Option 3**: You have manually changed the configuration file for your S3-BOX:
+  - You need to compile your firmware. Follow the steps in the [ESPHome documentation](https://esphome.io/guides/getting_started_command_line).
```

Comment:
I think this should be expanded a bit and not just dump people onto this page -- it quickly gets pretty technical. How about this?

```suggestion
  - You need to compile your own firmware. To do so, either:
    - Use the ESPHome dashboard add-on within Home Assistant. While the easiest option, it tends to be the slowest and may fail, particularly on older systems or on systems with limited memory/CPU resources.
    - Follow the steps in the [ESPHome documentation](https://esphome.io/guides/getting_started_command_line) and use a desktop-class system to compile and install the firmware. Initial setup is more complex, but the process is significantly faster and more reliable.
```